src/TouristInfoCards.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export default function TouristInfoCards(props) {
+  let card = props.card.map((el) => (
```

Comment:
Think this could be a `const`? Always better to use const where you dont need to  reassign a variable 😄